app/assets/stylesheets/modules/_sponsors.scss

```diff
@@ -1,44 +1,51 @@
-@mixin inline-image($background-image, $image-height, $image-width) {
-  display: inline-block;
-  height: $image-height;
-  width: $image-width;
+@mixin background-image($background-image) {
   background-image: image-url($background-image);
+  background-position: center center;
+  background-size: contain;
+  background-repeat: no-repeat;
 }
 
-.sponsor-row {
-  margin-top: 40px;
-  margin-bottom: 40px;
-
-  h3 {
-    margin-top: 0;
+.sponsors {
+  .sponsor-row {
```

Comment:
This is due to namespacing I talked about in the previous comment. Sponsors page's contents are wrapped inside the `.sponsor` class.